src/models/euDccCertificates/index.ts

```diff
@@ -64,11 +64,7 @@ const genEuDccCertificates = async (
       process.env.SIGNING_EU_QR_PRIVATE_KEY,
       ""
     );
-    const euDccGenerator = EuDccGenerator(
-      publicKey,
-      privateKey,
-      euSigner.issuer
-    );
+    const euDccGenerator = EuDccGenerator(publicKey, privateKey, "SG");
```

Comment:
Any reason for replacing `euSigner.issuer`?

Reply:
I did a rebase with master and I think those changes were showing in my branch as "my changes". 

Master branch: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/src/models/euDccCertificates/index.ts#L65

This branch: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/chore/improve-error-handling/src/models/euDccCertificates/index.ts#L67 (extra lines due to added try/catch wrapper)